apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/ValueConversion.cs

```diff
@@ -76,21 +76,16 @@ public static object ConvertToClrType(this Value wireValue, V1.Type spannerType,
                 return Convert.ToUInt16((long) ConvertToClrTypeImpl(wireValue, spannerType, typeof(long)));
             }
 
-            if (targetClrType == typeof(char))
+            if (targetClrType == typeof(sbyte))
             {
-                return Convert.ToChar((long) ConvertToClrTypeImpl(wireValue, spannerType, typeof(long)));
+                return Convert.ToSByte((long)ConvertToClrTypeImpl(wireValue, spannerType, typeof(long)));
```

Comment:
you mean why use Convert.ToSByte when you can just cast long to sbyte?  I wanted consistency in explaining to developers what to expect (expect convert to 'yourtype' to be called).
I dont see a downside to using Convert...

Agree with the extra helper that does the cast.